dptb/tests/test_dftb_scc.py

```
@@ -0,0 +1,1032 @@
+
+    # Verify run 1 and run 2 (reused instance) give identical results
+    assert torch.allclose(elec_totE_run1, elec_totE_run2, atol=1e-10), \
+        f"Reused instance elec_totE differs: {elec_totE_run1} vs {elec_totE_run2}"
+    assert torch.allclose(elec_H0_bandE_run1, elec_H0_bandE_run2, atol=1e-10), \
+        f"Reused instance elec_H0_bandE differs: {elec_H0_bandE_run1} vs {elec_H0_bandE_run2}"
+    assert torch.allclose(scc_shift_energy_run1, scc_shift_energy_run2, atol=1e-10), \
+        f"Reused instance scc_shift_energy differs: {scc_shift_energy_run1} vs {scc_shift_energy_run2}"
+    assert torch.allclose(scc_shift_run1, scc_shift_run2, atol=1e-10), \
+        f"Reused instance scc_shift differs: {scc_shift_run1} vs {scc_shift_run2}"
+    assert torch.allclose(Gamma_run1, Gamma_run2, atol=1e-10), \
+        f"Reused instance Gamma differs"
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)